src/components/TodoComponents/TodoList.js

```diff
@@ -1,2 +1,20 @@
-// your components will all go in this `component` directory.
-// feel free to change this component.js into TodoList.js
+import React from 'react';
+import Todo from './Todo';
+
+const TodoList = props => {
+    return(
+        <div>
+            {props.tasks.map(eachOneInMap => (
```

Comment:
You could use filter here to implement a basic search.  You'll get a bunch of chances to do that in the coming weeks.